droom-app/src/components/Home.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+
+
+//Styling the page
+
+const Div = styled.div `
+padding-top: 40px;
+margin-left: 20px;
+margin-right: 20px;
+
+
+
+`
+const Img = styled.img `
+object-fit: cover;
+width: 100%;
+height: 300px;
+border: 3px solid lavender;
+
+`
+
+
+const Button = styled.button `
+border: 3px solid lavender;
+border-radius: 20px;
+padding: 20px 20px 20px 20px;
+margin-top: 20px;
+color: white;
+background: #4389A2;
+font-size: 1rem;
+
+
+`
+
+
+
+export default function Home() {
+
+
+    return (
+
+        <div className="home-wrapper">
+        
+        <h1>Welcome to Droom</h1>
+        <p>...the place to find your dream job by swiping right</p>
+
+
+            <Div className="imagecontainer">
+            <Img className="home-image"
```

Comment:
You classname methodology should be consistent around the app. if you want to use camelCase use it all round, if you want to use dashes, use it all round too.